packages/tools/viewer-alpha/src/viewerElement.ts

```diff
@@ -559,6 +561,28 @@ export class HTML3DElement extends LitElement {
         this._viewerDetails?.viewer.toggleAnimation();
     }
 
+    /**
+     * Show the inspector for this scene
+     * @param options options passed through to Inspector.
+     */
+    public async showInspector(options?: IInspectorOptions): Promise<void> {
+        if (!this._inspectorInstance) {
+            await import("core/Engines/WebGPU/Extensions/engine.query");

```

Comment:
Is something in the Insepctor path supposed to import this automatically? If I don't have this, the stats part of inspector fails.